log/zapcore_handler.go

```diff
@@ -158,11 +158,6 @@ func emit(scope *Scope, level zapcore.Level, msg string, fields []zapcore.Field)
 	}
 }
 
-// backwards compatibility. TODO(mostrowski): remove this
-func (s *Scope) emit(level zapcore.Level, ps bool, msg string, fields []zapcore.Field) {
-	emit(s, level, msg, fields)
-}
-
```

Comment:
I went with removal instead of `nolint`-ing as the TODO mentions to remove. I'm not sure what might still require the function.